### ISSUE TITLE
'RunPython()' call reporting 'module' object has no attribute error in log file

### ISSUE THREAD

**Issue by rkpatel33** [ZoomerAnalytics/xlwings#657]
I created a test excel Workbook using:

`xlwings quickstart test`

And inside of the workbook module, I set up the following 2 subs:

```vb
Sub SampleCall()
    RunPython ("import test; test.demo()")
End Sub

Sub SampleCall2()
    RunPython ("import test; test.demo_with_caller()")
End Sub
```
And set up `test.py` as follows in the same directory as the excel file that was created by `quickstart`:

```python
import xlwings as xw

def demo():
    """
    Test call from vb using 'caller'.
    """
    wb = xw.Book.caller()
    wb.sheets['Sheet1'].range('a1').value = wb.sheets['Sheet1'].range('a1').value + 1

def demo_with_caller():
    """
    Test call from vb WITHOUT 'caller'.
    """
    xw.Book('test.xlsm').set_mock_caller()
    wb = xw.Book.caller()
    wb.sheets['Sheet1'].range('a1').value = wb.sheets['Sheet1'].range('a1').value + 1
```

When I click `Run Program` from the VB editor window to run each one, I do not get a message box alert with a stack trace, but the function is definitely failing. When I check the log file, I get the following tracebacks:

```shell
⇒  cat  ~/Library/Containers/com.microsoft.Excel/Data/xlwings_log.txt
Traceback (most recent call last):
  File "<string>", line 1, in <module>
AttributeError: 'module' object has no attribute 'demo_with_caller'
```

I'm not quite sure why the RunPython module can't find these functions inside the `test.py` module. When I run the commands from the python shell, they work fine:

![image](https://cloud.githubusercontent.com/assets/3799489/25162765/0d076d10-247a-11e7-82fb-ad90e88263cc.png)

And I see the expected side effect of the cell incrementing in the cell. Any idea on what could be wrong?

When I pull the demo fibonacci zip down, the `Lite` button works fine.

Here is the entire folder if helpful: https://www.dropbox.com/sh/l48yxpowlgsdyqv/AADjo9nCn0RBnI4Tk5uqf3PBa?dl=0

**Comment by fzumstein** [comment]
Note that `set_mock_caller` is not thought to be used inside a function that you call. Anyhow, the real issue will be that you should use the whole path of your excel file as the current working directory is not guaranteed.

you're right though about the missing error pop up. I ll open a seperate issue for that.

**Comment by rkpatel33** [comment]
Re you response on the excel file path - I'm a little confused on where I `should use the whole path of your excel file`. What point in the code are you referring to?

The only place I reference the file path is in `demo_with_caller()`, but the function `demo()` does not contain and excel file path and fails. Either way, the failure occurs well before that as both these functions are not found.

**Comment by fzumstein** [comment]
You can set it in the VBA settings: http://docs.xlwings.org/en/stable/vba.html#settings

**Comment by fzumstein** [comment]
Missing error pop up will be fixed with 0.10.5 see #660. Closing this as I expect you could fix the other part.
